tests/manager/manager-startSession_error-002.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+MongoDB\Driver\Manager::startSession() snapshot and causalConsistency cannot both be true
+--DESCRIPTION--
+Session spec prose test #1
+https://github.com/mongodb/specifications/blob/master/source/sessions/tests/README.rst#prose-tests
```

Comment:
I didn't link to the exact prose test here because https://github.com/mongodb/specifications/pull/1045 is still awaiting review. If that's merged before this, I'll revise this accordingly.

Reply:
The full fragment is https://github.com/mongodb/specifications/blob/master/source/sessions/tests/README.rst#setting-both-snapshot-and-causalconsistency-to-true-is-not-allowed, which is quite long, so I think I'll actually just leave this as-is.